src/styles/blocks/card.css

```diff
@@ -0,0 +1,100 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+  padding: 32px 16px 16px;
+  max-width: 200px;
+
+  color: #060b35;
+
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card_image {
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.card__title {
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+
+  margin-bottom: 4px;
+}
+
+.card__subtitle {
+  font-size: 10px;
+  line-height: 14px;
+  font-weight: 400;
+  color: #616070;
+
+  margin-bottom: 16px;
+}
+
+.card__rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  margin-bottom: 24px;
+}
+
+.card__review {
+  font-size: 10px;
+  line-height: 14px;
+  font-weight: 400;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+
+  margin-bottom: 16px;
+}
+
+.card__price-subtitle {
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 400;
+  color: #616070;
+}
+
+.card__price-value {
+  font-size: 16px;
+  line-height: 18px;
+  font-weight: 700;
+  color: #060b35;
+}
+
+.card__link {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 166px;
+  height: 40px;
+
+  text-transform: uppercase;
+  color: #fff;
+  text-decoration: none;
+
+  font-size: 14px;
+  line-height: 18px;
+  font-weight: 700;
+  background-color: #00acdc;
+  border: 1px solid transparent;
+  border-radius: 5px;
+}
+
+.card__link:hover {
+  background-color: #fff;
+  color: #00acdc;
+  border-color: #00acdc;
```

Comment:
Would be great, if you move all the colors to the specific variables

Reply:
done